sormas-e2e-tests/src/test/resources/features/sanity/web/PathogenTest.feature

```diff
@@ -0,0 +1,84 @@
+@UI @Sanity @Sample @PathogenTest
+Feature: Sample Functionalities
+
+  Scenario: Edit a new Sample
+    Given I log in with National User
+    And I click on the Cases button from navbar
+    And I click on the NEW CASE button
+    When I create a new case with specific data
+    And I collect the case person UUID displayed on Edit case page
+    And I click on New Sample
+    When I collect the sample UUID displayed on create new sample page
+    And I create a new Sample with specific data and save
+    And I click on the Sample button from navbar
+    And I search for Sample using Sample UUID from the created Sample
+    When I open created Sample
+    Then I check the created Sample is correctly displayed on Edit Sample page
+    When I change all Sample fields and save
+    Then I check the edited Sample is correctly displayed on Edit Sample page
+
+  Scenario: Add a Pathogen test from Samples and verify the fields
```

Comment:
I'm not sure if the PR is done based on the commented lines above, but please try to avoid making complex tests as you updated this one.
Better to create new ones with an isolated check instead :D